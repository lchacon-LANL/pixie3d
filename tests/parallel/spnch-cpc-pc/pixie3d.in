 &datin
    ilevel   = 0
    test     = t

    nxd      = 32 
    nyd      = 32 
    nzd      = 1

    xmax     = 1d0
    ymax     = 0.
    zmax     = 1.

    numerical_grid = f

    bcs      = 'spt','def','per','per','per','per'

    eta      = 1d-3
    nu       = 1d-3
    dd       = 0d-3
    chi      = 0d-2
    
    equil    = 'spnch'

    coords   = 'hel'

    dlambda  = 0.5
    gparams  = 1d0,-2d0,1d0,2d0,0d0

    gamma    = 1d0

    dt       = 1d0
    tmax     = 1d1
    dstep    = 1d1

    pvx      = 1d-4
    nh2      = 1
    odd      = f,f,f
    random   = f

    cnfactor = 0d0
    sm_flag  = 2

    restart  = f
    timecorr = f

    source   = t

    pc_type  = 's1'
    pc_tol   = 1d-1
    mg_vcyc  = 1
    sm_iter  = 3
    sm_zebra_relax=f
    mf_eps   = 1d-4
!    mg_coarse_size = 2
!    mgcf     = 8
!    mg_coarse_solve = f

    iguess   = 0

    maxksp   = 20
    maxitgm  = 20
    tolgm    = 8d-1
    rtol     = 1d-3
    atol     = 0d0
    method   = 1
    vol_wgt  = t

    solenoidal = t
    nc_eom_f   = t
    nc_eom_v   = f
    solve_rho  = t
    advect     = 4
 /

 &graphics
    sel_diag  = 2,12,15,17,18,19,20,22,23
    sel_graph = 1,2,3,5,6,7,9,10,11
    dplot = 1d-2
    prof_conf = 1,'r',1,10,1
    cont_conf = 3,'x','y',1,1,1
    hdf_plot = f
 /
