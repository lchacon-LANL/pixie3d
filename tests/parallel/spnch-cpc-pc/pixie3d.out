
  Grid mesh:................   32x   32x    1
  Number of grid levels:....    3,    4,    1
  Preconditioning method:...  s1
  Time correction...........   F
  External source...........   T
  Numerical grid............   F
  Scalar advection scheme...  SMART   
  Velocity advection scheme.  SMART   
  TS scheme................. BDF2     

   itime    time     dt     vx_max   vy_max   vz_max  GMRES  Newton  CN factor  GMRES-DC
      1    0.100  1.0E-01   8.9E-05  5.1E-05  2.9E-06    12     5     0.000        9
      2    1.000  9.0E-01   3.9E-05  1.9E-04  4.5E-05    22     5     0.000       10
      3    2.000  1.0E+00   1.4E-05  1.8E-04  6.8E-05    29     6     0.000       10
      4    3.000  1.0E+00   2.4E-05  1.5E-04  5.7E-05    16     5     0.000       10
      5    4.000  1.0E+00   3.8E-05  1.0E-04  3.2E-05    24     5     0.000       11
      6    5.000  1.0E+00   4.2E-05  1.1E-04  2.7E-05    14     5     0.000       10
      7    6.000  1.0E+00   3.4E-05  1.1E-04  3.9E-05    18     5     0.000       10
      8    7.000  1.0E+00   2.8E-05  1.3E-04  5.1E-05    20     5     0.000       10
      9    8.000  1.0E+00   3.0E-05  1.3E-04  5.1E-05    14     5     0.000       10
     10    9.000  1.0E+00   3.6E-05  1.2E-04  4.5E-05    18     5     0.000       10
     11   10.000  1.0E+00   4.0E-05  1.2E-04  4.5E-05    20     5     0.000       10

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   11       5.1       18.8        3.7
